script.js

```diff
@@ -0,0 +1,25 @@
+const crossButton = document.querySelector('.overlay-cross-button');
+const mobileMenuOverlay = document.querySelector('.mobile-menu-overlay');
+const navHamIcon = document.querySelector('.navigate');
+const mobileMenuLinks = document.querySelectorAll('.list-item > a');
+
+function hideMobileMenu() {
+  requestAnimationFrame(() => {
+    mobileMenuOverlay.style.left = '100%';
+  });
+}
+
+navHamIcon.addEventListener('click', () => {
+  mobileMenuOverlay.style.left = '0%';
+  mobileMenuOverlay.style.display = 'flex';
+});
+
+crossButton.addEventListener('click', () => {
+  hideMobileMenu();
+});
+
+mobileMenuLinks.forEach((mobileMenuLink) => {
```

Comment:
- [ ] Having made a great effort working on the mobile menu 👏 👏, I noticed some strange functionality in your work, when the `hamburger` icon is clicked the mobile menu should pop out over the `main page` but in your case, the mobile menu is coming out by the side of the `main page` and the `(X)` icon is not working for closing the mobile menu.

  - Kindly check the video below to see the aspect of the project am talking about 👇 

[screen-capture (2).webm](https://user-images.githubusercontent.com/78634841/210308427-b10e78f8-4abf-46f3-8d57-99a4207aff0c.webm)

> N.B :  ![annie](https://user-images.githubusercontent.com/78634841/210308622-beae90cc-31bf-4a3a-975a-25979f57a663.png)